core/src/components/item/item.tsx

```diff
@@ -121,6 +126,8 @@ export class Item {
       ...getElementClassMap(this.el.classList),
       'item-disabled': this.disabled,
       'item-detail-push': showDetail,
+      [`item-lines-${this.lines}`]: this.lines,
```

Comment:
While this probably works fine, I think it's a good idea to cast "this.lines" explicitly to boolean, future updates (strict typing) might detect this as invalid, since CssClassMap states a map of `[string]: boolean`

```
      [`item-lines-${this.lines}`]: !!this.lines,
```